reflex/.templates/jinja/web/utils/context.js.jinja2

```diff
@@ -25,10 +25,20 @@ export const clientStorage = {}
 
 {% if state_name %}
 export const state_name = "{{state_name}}"
-export const onLoadInternalEvent = () => [
-    Event('{{state_name}}.{{const.update_vars_internal}}', {vars: hydrateClientStorage(clientStorage)}),
-    Event('{{state_name}}.{{const.on_load_internal}}')
-]
+export const onLoadInternalEvent = () => {
```

Comment:
Nice, so we avoid sending the event if no cookies are set, is that right? Can we add some comments to this explaining high level